XamarinCommunityToolkit/Views/AvatarView.shared.cs

```diff
@@ -179,22 +189,29 @@ void OnSizePropertyChanged()
 			BatchCommit();
 		}
 
-		void OnValuePropertyChanged()
+		async void OnValuePropertyChanged()
 		{
 			if (Control == null)
 				return;
 
 			Image.BatchBegin();
-			var source = Source;
-			Image.IsVisible = source != null;
-			Image.Source = source;
+			Image.IsVisible = false;
+			var imageBytes = await GetImageBytesAsync(Source);
```

Comment:
Well, 

1) OnValuePropertyChanged is called multiple times, and multiple times we run downloading task
2) We lost data caching

Reply:
As for the first item it is easy fix.
How to solve the issue with caching? I found that FFImageLoading has all the features we need, but it is very complex and I don’t want to add a new dependency

Reply:
Probably I have found a solution which should satisfy our requirements. And it should work with net standard 1.0